imports/plugins/core/navigation/client/components/NavigationDashboard.js

```diff
@@ -3,7 +3,7 @@ import PropTypes from "prop-types";
 import { Dialog, DialogContent } from "@material-ui/core";
 import Button from "@reactioncommerce/catalyst/Button";
 import HTML5Backend from "react-dnd-html5-backend";
-import { DragDropContext } from "react-dnd";
+import { DndProvider } from "react-dnd";
```

Comment:
This import seems unused. Can it be removed?

Reply:
Good catch! A leftover from my debugging process... Thanks for that!